SavedSettings.cs

```diff
@@ -61,8 +72,16 @@ internal void LoadSettings()
                 try
                 {
                     _priorytyBuildings.Clear();
+                    _globalBlueprints.Clear();
+
                     var json = File.ReadAllText(savePath);
                     JsonConvert.PopulateObject(json, this);
+
+                    if (!GlobalBlueprints.Any())
+                    {
+                        GlobalBlueprints.Add(new GlobalBlueprint("Default"));

```

Comment:
Hello..
You cannot add it to variable initialisation? 
Here https://github.com/ingwarsw/NGU-Industries-Injector/pull/5/files#diff-d04f4110fbc2ca2952d99a5550658f9b7f1c89401a84d80da80b5c4065aac459R17

Reply:
I did try that initially but I ran into a problem I can't remember the specific details of anymore, I think SaveSettings wasn't working via getter or something along those lines. After fixing that problem I neglected to go back and try initialization again. I just tested it and it works fine now

Commit: https://github.com/wentimo/NGU-Industries-Injector/commit/cddbce61401382f86e0693f96b075aac860e1035

That commit will be part of the next PR, or I can make a small PR with just that commit if you'd prefer that

Reply:
Any feature ideas you'd like to see implemented? I was thinking about adding functionality for using the optimizer tool (https://github.com/plasma119/NGU-INDUSTRIES-optimizer) without running AHK (https://github.com/NotThat/NGU-AHK) . With that functionality in the injector, using the optimizer would simply be clicking on "Copy to clipboard" and then pressing F2 or some other hotkey to copy the setup from clipboard into NGU Industries